### ISSUE TITLE
Inconsistend layout of Lookup windows causes confusion

### ISSUE THREAD

**Issue by n8willis** [fontforge/fontforge#4590]
The window layouts for OpenType GSUB/GPOS lookups are inconsistent, which causes major confusion when inspecting a font's features.

Specifically, in some of the windows, the left-hand column shows the input sequence for the lookup and the right-hand column shows the output sequence, while in other windows, the left-hand column shows the OUTPUT sequence for the lookup and the right-hand column shows the INPUT.

This leads to situations like the following, where two lookups have inverse effects but are presented virtually identically: 
![ff-window-1](https://user-images.githubusercontent.com/2192132/105580321-98c3cd80-5d83-11eb-9070-fb89f0b63ea2.png)
![ff-window-2](https://user-images.githubusercontent.com/2192132/105580322-9bbebe00-5d83-11eb-8217-2e5121414d92.png)

The layout ought to be consistent to begin with as a matter of principle; the fact that it also has the detrimental side-effect of creating confusion just compounds the importance.

### Important
Mark with [x] to select.

**Comment by frank-trampe** [comment]
I agree that we ought to standardize upon one ordering. Any suggestion as to which one?

**Comment by n8willis** [comment]
Well, assuming that the language of the UI is LTR, I think `input -> output` is kinda the only option. Better match for FEA file syntax, in addition to traditional logical expressions.

A little less clear how to handle the reverse chaining contextual lookup type, but for simple stuff.

**Comment by dscorbett** [comment]
Currently, the first column contains a single glyph and the second column may contain multiple glyphs. That is why the input/output order is inconsistent. If the order is made consistent, the first column may contain multiple glyphs; if so, will it be possible to see the second column’s contents without scrolling to the right?

**Comment by ctrlcctrlv** [comment]
I agree with `input` → `output`.

**Comment by ctrlcctrlv** [comment]
@dscorbett There is, (or should be,) a way to expand the table based on its contents in GDraw. I think it's done automatically, but would need to look through the source for certainty.

**Comment by skef** [comment]
@ctrlcctrlv That's potentially the problem, though. If there are five glyphs (for example) in an entry in the "input" column and you auto-size on it it could fill up the whole area. Avoiding that problem means giving a sensible setting to the sizing parameters and making them authoritative. (Much of the code still has sizing parameters for the dialog features but they're mostly ignored (as far as I can tell) in favor of calculations based on content.)
